app/views/home/index.html.erb

```diff
@@ -30,7 +30,7 @@
         <div class="float-left mr-4 float-md-none mr-md-0 mb-md-3"><img src="https://assets-cdn.github.com/images/modules/site/icons/opensource-ico-best.svg?sn" alt="" aria-hidden></div>
         <div class="overflow-hidden">
           <h3 class="alt-h4 mb-2">Learn from the best in the&nbsp;field</h3>
-          <p class="alt-text-small text-gray">Amazing developers around the world participate in open source. Contribute code to projects that run the world.</p>
+          <p class="alt-text-small text-gray">Developers around the world participate in open source. Contribute code to projects that run the world.</p>
```

Comment:
I like this change. Is it worth going even further to say something like "`Developers of all levels of experience`?